frontend/src/components/data-products/data-output-form/databricks-data-output-form.component.tsx

```diff
@@ -12,29 +12,52 @@ type Props = {
 
 export function DatabricksDataOutputForm({ form, identifiers, external_id, sourceAligned }: Props) {
     const { t } = useTranslation();
+    const entireSchema = Form.useWatch('entire_schema', form);
     let databaseOptions = (identifiers ?? []).map((database) => ({ label: database, value: database }));
-
+    const databaseValue = Form.useWatch('schema', form);
+    const suffixValue = Form.useWatch('schema_suffix', form);
+    const tableValue = Form.useWatch('table', form);
     useEffect(() => {
         let databaseOptionsList = identifiers //TODO
         if (!sourceAligned) {
             databaseOptionsList = [external_id]
-            form.setFieldsValue({ database: external_id});
+            form.setFieldsValue({ schema: external_id});
         } else {
-            form.setFieldsValue({database: undefined})
+            form.setFieldsValue({schema: undefined})
         }
         databaseOptions = (databaseOptionsList ?? []).map((database) => ({ label: database, value: database }));
     }, [sourceAligned]);
 
+
+    useEffect(() => {
+        let result = databaseValue;
+        if (databaseValue){
+            if (suffixValue) {
+                result += `__${suffixValue}`;
```

Comment:
Identify a DBX output using catalog.schema.table
```suggestion
                result += `.${suffixValue}`;
```